code/optimize/src/androidTest/java/com/adobe/marketing/mobile/optimize/OptimizeFunctionalTests.java

```diff
@@ -76,6 +76,33 @@ public void testExtensionVersion() {
         Assert.assertEquals(OptimizeTestConstants.EXTENSION_VERSION, Optimize.extensionVersion());
     }
 
+    @Test
+    public void testUpdatePropositions_timeoutError() throws Exception {
+        // Setup
+        final String decisionScopeName = "decisionScope";
+        Map<String, Object> configData = new HashMap<>();
+        configData.put("edge.configId", "ffffffff-ffff-ffff-ffff-ffffffffffff");
+        updateConfiguration(configData);
+
+        // Action
+        Optimize.updatePropositions(
+                Collections.singletonList(new DecisionScope(decisionScopeName)),
+                null,
+                null,
+                new AdobeCallbackWithError<Map<DecisionScope, OptimizeProposition>>() {
+                    @Override
+                    public void fail(AdobeError adobeError) {
```

Comment:
I guess this should fail with AEPOptimizeError and not Adobe Error. @ishita-gambhir-adobe can you check this once and thus add assertions for status, title, detail for AEPOptimizeError in that case.

Reply:
Done!